lib/modules/manager/flux/readme.md

```diff
@@ -36,16 +36,18 @@ If `tag` _and_ `digest` exist then `digest` takes precedence.
 
 ### Flux system manifests support
 
-Support updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+Renovate supports updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+System manifests are identified via being named `gotk-components.y(a)ml` and containing comment headers as generated by `flux bootstrap`.
 
```

Comment:
Just a comment to say that you should update your branch to match the current `main`, because we just merged in a bunch of `v36` breaking changes. 😉

Reply:
Aah ok, I'll merge in `main`